src/wrap.cr

```diff
@@ -4,10 +4,17 @@ module Wrap
     lines = [] of String
     until chars.empty?
       line = chars.shift(width + 1)
```

Comment:
I know this line isn't part of the PR, but after doing some local testing of this branch I noticed it was causing problems. for example, sometimes the last letter of a line is pushed to the next line:

![screenshot 2018-09-04 08 53 18](https://user-images.githubusercontent.com/385726/45042648-0be81000-b020-11e8-9d72-f9259c3f5721.png)

If we make this line

```crystal
line = chars.shift(width)
```

that bug disappears. Is there a reason why you originally had the `+1` in there?

Reply:
I don't remember! LOL If the specs pass without the +1, sounds good to me.

Reply:
Yeah I will a bit later.

Reply:
@geolessel This is done.